components/NewsCard.jsx

```diff
@@ -45,7 +56,7 @@ export default function NewsCard({ journalist, news }) {
 
         <View style={ styles.timeStampContainer }>
           <Text style={ globalStyles.timeStamp } >
-            { news.timestamp }
+            { formatPublishedTime(article.published_time) } {/* Format the published time */}
```

Comment:
it might be pænere at have det her som et kald til en variabel og så bare sætte variablen ind her. u know readability